src/pages/BuySell/BuySell.tsx

```diff
@@ -0,0 +1,220 @@
+      let filteredResults: any[] = []
+
+      filteredResults = results.map(result => {
+        var i = filteredResults.findIndex((x: { ticker: any }) => x.ticker === result.ticker)
+        if (i <= -1) {
+          return result
+        }
+        return undefined
+      })
+
+      return filteredResults.filter(x => !!x)
+    },
+    []
+  )
+
+  const ridirectToGem = () => {
+    setRedirectToGem(true)
+  }
+
+  //Fetch supported coins and set them to state
+  const fetchSupportedCoins = useCallback(async () => {
+    try {
+      //const res = await getCoinifySupportedCurrencies();
+      const coinifyList = await getCoinifySupportedCurrencies()
+      const wyreList = await getWyreSupportedCurrencies()
+      const buyList = filterAndMerge(coinifyList, wyreList, 'destination', buyFilter)
+      const sellList = filterAndMerge(coinifyList, wyreList, 'source', sellFilter)
+      setSupportedCoins(buyList)
+      setSupportedSellCoins(sellList)
+    } catch (e) {
+      console.error(e)
+    }
+  }, [filterAndMerge])
+
+  useEffect(() => {
+    fetchSupportedCoins()
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
+
+  return (
+    <>
+      {redirectToGem && (
+        <Redirect
+          to={{
+            pathname: '/buysell/gem',
+            state: { buyCoinList: supportedCoins, sellCoinList: supportedSellCoins }
+          }}
+        />
+      )}
+      {!redirectToGem && (
+        <Page>
+          <Flex
+            position={'absolute'}
+            top='50%'
+            left='50%'
+            marginRight={'-50%'}
+            transform={'translate(-35%,-50%)'}
+          >
+            <Card
+              textAlign='center'
+              alignSelf={'center'}
+              boxShadow='none'
+              borderWidth={0}
+              minWidth={'60%'}
+              maxWidth={isLargerThan550 ? '70%' : '120%'}
+            >
+              <Card.Header>
+                <Card.Heading>
+                  <Text translation='buysell.page.title' />
+                </Card.Heading>
+              </Card.Header>
+              <Card.Body>
+                <Text lineHeight={1.2} translation='buysell.page.titleMessage' />
+                <Flex
+                  flexDirection={'column'}
+                  justifyContent={'flex-start'}
+                  mt={'5%'}
+                  alignItems={'flex-start'}
+                >
+                  <Box>
+                    <Button
+                      variant='ghost'
+                      onClick={() => {
+                        ridirectToGem()
+                      }}
+                      isActive={true}
+                      justifyContent='flex-start'
+                      _focus={{
+                        shadow: 'outline-inset'
+                      }}
+                      p={'25px'}
+                      wordBreak={'break-word'}
+                      minWidth={isLargerThan550 ? '120%' : '100%'}
```

Comment:
i'm going to stop commenting on each of these, but no hardcoded values, we have set sizes/breakpoints and should be using those, @reallybeard can advise